entrypoint.sh

```diff
@@ -0,0 +1 @@
+poetry run gunicorn --bind 0.0.0.0:$PORT 'todo_app.app:create_app()'
```

Comment:
It's a good habit to always include a [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) to help the system know what sort of script this is, e.g.
```suggestion
#!/bin/bash
poetry run gunicorn --bind 0.0.0.0:$PORT 'todo_app.app:create_app()'
```